src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Tree/Dependencies/Models/TargetDependencyViewModel.cs

```diff
@@ -16,15 +16,15 @@ internal sealed class TargetDependencyViewModel : IDependencyViewModel
 
         public TargetDependencyViewModel(TargetFramework targetFramework, DiagnosticLevel diagnosticLevel)
         {
-            Caption = targetFramework.ShortName;
+            Caption = targetFramework.TargetFrameworkMoniker;
```

Comment:
If this truly is the TFM, I would expect the display string to change from `net472` to `.NETFramework,Version=v4.7.2`, which probably isn't what we want across all target frameworks. Am I missing something?